### DISCUSSION TITLE
how to add state/province boundaries?

### DISCUSSION THREAD

**Discussion by opierpaoli** [raphaelquast/EOmaps#115] [Q&A] [answered]
Hi, 

How do I add state/province boundaries to a map? The code below works for countries, but I can't seem to add the layer from this website https://www.naturalearthdata.com/features/ which contains state boundaries.

`m.add_feature.cultural_50m.admin_0_countries()`

Thanks!

**Comment by raphaelquast** [comment, marked as answer]
Hey, I guess you've picked the wrong scale-rank.... did you check the `admin_1_countries` dataset?

```python
from eomaps import Maps
m = Maps()
m.add_feature.cultural_10m.admin_1_states_provinces(fc="none", ec="k", lw=0.25)
```

![grafik](https://user-images.githubusercontent.com/22773387/192459757-4559c73e-b212-40d9-958a-75c1823f8cc1.png)
